### PR TITLE
Documentation for regionendpoint config parameter

### PR DESCRIPTION
**Pull request by legionus**
Fixes https://github.com/openshift/image-registry/issues/104

@bparees please review.

### CODE REVIEW

File: install_config/registry/extended_registry_configuration.adoc [hunk 1]
```diff
@@ -436,6 +436,61 @@ more efficient registry operation. On the other hand, memory usage will raise
...
+----
+
+[NOTE]
+====
+It is very important in this case that the `region` and `regionendpoint` fields
+are consistent between themselves. Otherwise the integrated registry will start,
+but it can not read or write anything to the S3 storage.
+====
+
+The `regionendpoint` can also be useful if you use a different from the
```
Comment (bparees) [inline install_config/registry/extended_registry_configuration.adoc]: "if you use a different from" - there's a word missing here but i'm not sure what it is.

**Comment by bparees** [comment]
one nit and lgtm

File: install_config/registry/extended_registry_configuration.adoc (hunk 1 above)
Comment (legionus) [reply install_config/registry/extended_registry_configuration.adoc]: Fixed.

**Comment by bparees** [comment]
@openshift/team-documentation bump

**Comment by bparees** [comment]
@bmcelvee this one isn't 3.11 specific so i think it can go in.

**Comment by bmcelvee** [comment]
Thanks, @bparees!

**Comment by bmcelvee** [comment]
@dongboyan77 PTAL

**Comment by bmcelvee** [comment]
Overall, LGTM. Once the QE review is finished, I'll merge this PR. Since it isn't 3.11 specific, is it applicable for 3.10?

**Comment by bparees** [comment]
> Overall, LGTM. Once the QE review is finished, I'll merge this PR. Since it isn't 3.11 specific, is it applicable for 3.10?

yes.  I think it goes back as long as this section has existed.

**Comment by vikram-redhat** [comment]
@dongboyan77 - can you PTAL? @xltian

**Comment by dongboyan77** [comment]
LGTM, sorry for review delay

**Comment by bmcelvee** [comment]
In looking at this one a bit more, I would like to go ahead and merge this PR, then move the applicable content into the existing "Use Amazon S3 as a Storage Back-end" (https://docs.openshift.com/container-platform/3.9/install_config/registry/deploy_registry_existing_clusters.html#registry-amazon-s3-storage-back-end) section. There is quite a bit of overlap between the content introduced in this PR and the existing section, so it looks like a natural fit. 

Do you think that would be an acceptable path forward?

**Comment by bparees** [comment]
The two sections definitely need to be reconciled, but i think i'd go the other way.

i'd leave:
https://docs.openshift.com/container-platform/3.9/install_config/registry/deploy_registry_existing_clusters.html#registry-amazon-s3-storage-back-end

as a high level overview of what can be configured, and have it link out to:

https://docs.openshift.com/container-platform/3.9/install_config/registry/extended_registry_configuration.html#docker-registry-configuration-reference-storage

for specifics about how to define said configuration and things to consider for the different storage types.

The challenge is that the first section is more focused on the oc commands you'd run to configure/deploy the registry, and the second section is more focused on the actual config definition/files.

**Comment by bmcelvee** [comment]
I pulled this commit into a new follow-up PR: https://github.com/openshift/openshift-docs/pull/11040. Tracking changes there.

**Pull request closed without merging**